sdk/resourcemanager/azure-resourcemanager-compute/src/test/java/com/azure/resourcemanager/compute/VirtualMachineManagedServiceIdentityOperationsTests.java

```diff
@@ -37,9 +37,10 @@ protected void cleanUpResources() {
     }
 
     @Test
-    @LiveOnly
+    @DoNotRecord(skipInPlayback = true)
     public void canSetMSIOnNewOrExistingVMWithoutRoleAssignment() throws Exception {
         // LiveOnly because test needs to be refactored for storing/evaluating PrincipalId
+        // and "test timing out after latest test proxy update"
         // Create a virtual machine with just MSI enabled without role and scope.
```

Comment:
We are seeing more and more of this "test timing out after latest test proxy update". What are the cause of this time out? Is it test proxy problem, or our TestBase problem?

Reply:
@weidongxu-microsoft 
At present, I have found that when deserializing large lists and complex structure lists during CI testing, timeout exceptions will occur in almost all test environments. However, these test cases can successfully executed for PLAYBACK at location environment. So I think this is not just a problem with the test proxy or TestBase.